src/collector/facet_collector.rs

```diff
@@ -37,7 +37,10 @@ impl<'a> PartialOrd<Hit<'a>> for Hit<'a> {
 
 impl<'a> Ord for Hit<'a> {
     fn cmp(&self, other: &Self) -> Ordering {
-        other.count.cmp(&self.count)
+        match other.count.cmp(&self.count) {
```

Comment:
You can chain orderings using [`Ordering::then`](https://doc.rust-lang.org/std/cmp/enum.Ordering.html#method.then).

Reply:
oh cool! Thanks for the feedback. I thought there has to be a better way of doing this. will make changes in accordance with this.